### ISSUE TITLE
Nvidia support?

### ISSUE THREAD

**Issue by Miusaky** [vaxerski/Hyprland#12]
I couldn't find an answer to this in the wiki. Do you plan - if it doesn't as is - to add a flag or some level of support for running Hyprland with the propriertary Nvidia drivers? Since this seems to be based on wl-roots it is perhaps possible to add a flag like Sway has --my-next-gpu-wont-be-nvidia? 

Since Nvidia do not open source their drivers I'd understand if this wasn't something you wanted to do - in which case I'll wait until I can replace the card with an AMD one to use your WM. 

Last but not least - thank you for developing Hypr. I've been using it for a while and it's perfection. Looking forward to testing Hyprland as well.

**Comment by vaxerski** [comment]
nvidia support? sir I can't even get GDK popups to work yet.

I'll focus on this being able to become an actual stable thing before I add in any sort of nvidia support.

Afaik, sway's --unsupported-gpu flag is sway-made, not WLR. However, sway has most likely many patches in place to even get nvidia to work at all. 

If you try wms like river, dwl or vivarium, they most likely will just not work as well.

So, TLDR: definitely not a priority, probably not soon, questionable whether by me (as I do not own an nvidia gpu, sold all these pieces of garbage) but maybe someone will make a PR or at least help test

**Comment by vaxerski** [comment]
If there is no patches, hyprland should run without issues too. I have not made anything anti-nvidia.
